scss/_utilities_layout.scss

```diff
@@ -3,13 +3,21 @@
     @extend %fixed-width-container;
   }
 
+  // stylelint-disable selector-max-type -- table elements can use selector types
   @if ($table-layout-fixed) {
```

Comment:
While we are updating this, I'd remove the if statement. There is no huge benefit of having only one of those utils in the compiled version of Vanilla (they are tiny and will compress well). And it's just confusing for usage and documentation to have only one of the two available.

So let's just always have both utils available regardless of what is the default.

Reply:
If statement removed from PR and documentation updated accordingly